tests/e2e/chains/ethereum/Makefile

```diff
@@ -1,7 +1,7 @@
 GETH_HTTP_PORT ?= 8546
 BEACON_HTTP_PORT ?= 19596
-GETH_VERSION ?= v1.14.11
-LODESTAR_VERSION ?= v1.23.0
+GETH_VERSION ?= v1.14.12
```

Comment:
The latest version of geth is v1.15.x, which is the only officially supported version of pectraHF, so it might be a good idea to upgrade to the latest version.

Reply:
@toshihiko-okubo FYI, the following fix was also required for geth to work.
https://github.com/datachainlab/cosmos-ethereum-ibc-lcp/pull/74/commits/f0736af5b023961006b802c704403335508a2e24